api-reference/_hidden/dxGanttHeaderFilter/searchTimeout.md

```diff
@@ -5,7 +5,6 @@ default: 500
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a delay in milliseconds between typing a search string and the search execution.
```

Comment:
Do you mean a delay between the last change in the search box and the effect of that change?

Reply:
Yes.